_overviews/scala3-book/domain-modeling-fp.md

```diff
@@ -49,20 +47,20 @@ An FP design is implemented in a similar way:
 In this chapter we’ll model the data and operations for a “pizza” in a pizza store.
 You’ll see how to implement the “data” portion of the Scala/FP model, and then you’ll see several different ways you can organize the operations on that data.
 
-
-
 ## Modeling the Data
 
 In Scala, describing the data model of a programming problem is simple:
 
 - If you want to model data with different alternatives, use the `enum` construct
 - If you only want to group things (or need more fine-grained control) use `case` classes
 
-
 ### Describing Alternatives
 
 Data that simply consists of different alternatives, like crust size, crust type, and toppings, is concisely modeled with the Scala 3 `enum` construct:
 
+{% tabs data_1 %}
+{% tab 'Scala 3 only' for=data_1 %}
+
 ```scala
```

Comment:
there should be Scala 2 versions here using sealed trait and case object

Reply:
i dont know how to do it.